ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/SubmissionValidator.scala

```diff
@@ -122,6 +122,10 @@ class SubmissionValidator[LogResult](
       ) => Future[T],
   )(implicit logCtx: LoggingContext): Future[Either[ValidationFailed, T]] =
     Envelope.open(envelope) match {
+      case Right(_: Envelope.BatchMessage) =>
+        Future.successful(
+          Left(ValidationFailed.ValidationError("Validation of batches is not supported.")))
```

Comment:
Has this a chance to happen or only if it is a bug? If so we should probably log an error

Reply:
This is here only temporarily until the validator implementation lands in another PR.